stdlib/private/StdlibCollectionUnittest/MinimalCollections.swift

```diff
@@ -83,6 +83,8 @@ public enum UnderestimatedCountBehavior {
 ///
 /// This sequence is consumed when its generator is advanced.
 public struct MinimalSequence<T> : Sequence, CustomDebugStringConvertible {
+  public typealias Element = T
```

Comment:
Here it might be worth just renaming the generic parameter T to Element, for readability in any case